app/views/assessments/_gradesheet.html.erb

```diff
@@ -50,14 +49,16 @@
 <tr class="<%= aud_special_grade_type?(s.aud) && @cud.instructor? ? "special_submission" : "" %>">
   <td class="enumerator"></td>
   <td class="id" data-submission-id="<%= s.id.to_s %>">
-        <a tabindex="-1" class="email" href="#"><%= s.course_user_datum.email %></a>
+        <%= s.course_user_datum.last_name %>, <%= s.course_user_datum.first_name %>
```

Comment:
It'd maybe be nice to make this ordering (Last, First vs First Last, etc.) configurable somehow, but it's probably fine when used with the search feature for now.